src/Microsoft.Framework.PackageManager/Publish/DependencyContext.cs

```diff
@@ -68,9 +71,13 @@ public static FrameworkName GetFrameworkNameForRuntime(string runtime)
             {
                 case "mono":
                 case "clr":
-                    return VersionUtility.ParseFrameworkName("dnx451");
+                    // CLR currently supports anything <= dnx46
+                    return availableFrameworks
```

Comment:
One. Right here. But I get your point. I think it might be necessary to centralize the logic that determines the set of allowed frameworks for a specific DNX.